tests/query/highlights/fusion/basic.fusion

```diff
@@ -0,0 +1,61 @@
+include: SomeFile.fusion
+//<- include
+//     ^punctation.delimiter
+//       ^uri
+
+namespace ns = Neos.Fusion.Space
```

Comment:
very nice! Tests! It also helps us to see how the highlighting would look like. It looks awesome :tada: !

Reply:
Thank u - In my opinion it also makes is a lot easier to write the grammar or queries. At least, if you do it the first time like me :)

Reply:
Also for me when I did the Xth time.